Cargo.toml

```diff
@@ -11,5 +11,11 @@ readme = "README.md"
 documentation = "https://georust.github.io/rust-geojson/"
 keywords = ["geojson", "gis", "json", "geo"]
 
+[features]
+default = ["rustc-serialize"]
+with_serde = ["serde","serde_json"]
```

Comment:
So because I'm using `serde` and `serde_json` I can't just use `serde` as a feature name (otherwise it won't include `serde_json` too). Do you have any thoughts on getting around that?